pillar/top.sls

```diff
@@ -210,12 +210,17 @@ base:
     - mysql.devstack
     - elasticsearch.devstack
     - rabbitmq.devstack
-  'roles:rabbitmq':
+  'G@roles:rabbitmq and P@environment:(mitx-production|production-apps)':
```

Comment:
You can just move this line directly above the datadig integration line and remove the duplicated block